tests/queue/caliper-test_compute_fire.sh

```
@@ -0,0 +1,96 @@
+assert "no state file -> exit 1"     '[[ $RC -eq 1 ]]'
+
+# --- epoch mode ---
+tgt=$(( (now/3600 + 3)*3600 + 12*60 ))                      # future :12 local
+run --epoch "$tgt"
+assert "epoch mode exits 0"               '[[ $RC -eq 0 ]]'
+assert "epoch mode reports MODE=epoch"    '[[ "$(field MODE)" == "epoch" ]]'
+assert "epoch minute honored"             "[[ \"\$(field CRON | cut -d' ' -f1)\" -eq $(lmin "$tgt") ]]"
+
+# sub-minute future target bumps to the next whole minute (cron is minute-granular)
+nowx="$(date +%s)"; run --epoch "$(( nowx + 20 ))"
+assert "sub-minute target exits 0"        '[[ $RC -eq 0 ]]'
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)